test/integration/service_manual_guide_test.rb

```diff
@@ -37,15 +37,15 @@ class ServiceManualGuideTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "displays a summary if present" do
+  test "displays the description for a point" do
     setup_and_visit_example('service_manual_guide', 'point_page')
 
     within('.page-header__summary') do
       assert page.has_content?('Research to develop a deep knowledge of who the service users are')
     end
   end
 
-  test "the lede is not visible unless there is a summary" do
+  test "does not display the description for a normal guide" do
     setup_and_visit_example('service_manual_guide', 'service_manual_guide')
 
     refute page.has_css?('.lede')
```

Comment:
I *think* this might need updating since the header change went out. Might've missed it…